index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+
+const PORT = 1234;
+
+var data = require('./package.json');

```

Comment:
Rename the var to something more meaningful.

Reply:
ok